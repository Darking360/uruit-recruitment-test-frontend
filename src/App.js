import React from 'react';
import Spinner from './components/Spinner';
import GameSetup from './containers/GameSetup';
import styled from 'styled-components';

const AppContainer = styled.section`
  background-color: #333;
  width: 100vw;
  min-height: 100vh;
  height: 100%;
  color: white;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`;

// Add state to manage current screen

function App() {
  return (
    <AppContainer className="App">
      <Spinner width="10%" />
      <GameSetup />
    </AppContainer>
  );
}

export default App;
