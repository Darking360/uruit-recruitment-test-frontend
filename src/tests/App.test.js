import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('App', () => {

   it('renders without crashing', () => {
      shallow(<App />);
    });

    it('matches snapshot', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
    });

});