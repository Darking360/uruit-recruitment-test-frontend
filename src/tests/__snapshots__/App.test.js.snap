// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<styled.section
  className="App"
>
  <styled.div>
    <GameSetup
      setGame={[Function]}
    />
    <Styled(styled.button)
      onClick={[Function]}
    >
      Leader Boards
    </Styled(styled.button)>
  </styled.div>
</styled.section>
`;
